### PR TITLE
Update sqlclient-driver-support-lifecycle.md

### PR DESCRIPTION
**Pull request by ErikEJ**
Update with latest releases

### CODE REVIEW

**Comment by Court72** [comment]
@David-Engel

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/connect/ado-net/sqlclient-driver-support-lifecycle.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ New stable (GA) releases are published every six months on a regular cadence beg
 
 | Version | Official Release Date | Latest Patch Version | Patch Release Date | Support Level | End of Support |
 |--|--|--|--|--|--|
-| 5.2 | February 28, 2024 | - | - | Current | - |
-| 5.1 | January 19, 2023 | 5.1.5 | January 29, 2024 | LTS | January 20, 2026 |
-| 4.0 | November 18, 2021 | 4.0.5 | January 9, 2024 | LTS | November 19, 2024 |
+| 5.2 | February 28, 2024 | 5.2.2 | August 27, 2024 | Current | - |
+| 5.1 | January 19, 2023 | 5.1.6 | August 27, 2024 | LTS | January 20, 2026 |
+| 4.0 | November 18, 2021 | 4.0.6 | August 21, 2024 | LTS | November 19, 2024 |
 | 3.1 | March 30, 2022 | 3.1.5 | January 9, 2024 | LTS | March 30, 2025 |
```
Comment (David-Engel) [inline docs/connect/ado-net/sqlclient-driver-support-lifecycle.md]:
```suggestion
| 3.1 | March 30, 2022 | 3.1.7 | August 20, 2024 | LTS | March 30, 2025 |
```

**Review by David-Engel — APPROVED** [review]
Thanks for the edit! Add the update for 3.1 and I'll merge.

**Comment by Court72** [comment]
@ErikEJ - Can you respond to the requested changes or let us know if you would like us to commit them?

@David-Engel - When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.

**Comment by ErikEJ** [comment]
Please commit them.

**Comment by David-Engel** [comment]
#sign-off

**Pull request merged**